### PR TITLE
added map page

### PR DESCRIPTION
**Pull request by faesalm**
Added basic map page. Will be adding more features tonight/tomorrow

### CODE REVIEW

File: src/main/webapp/js/map_loader.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+let map;
+function createMap(){
```
Comment (alanensastegui) [inline src/main/webapp/js/map_loader.js:2]: For this function I think your method comment should only need to mention that the map is placed in a map element, and that is has an event listener

File: src/main/webapp/js/navigation-loader.js [hunk 2]
```diff
@@ -55,6 +55,15 @@ function addPublicFeed() {
   navigationElement.appendChild(createListItem(createLink('/feed.html', 'Public Feed')));
 }
 
+function addMap() {
```
Comment (alanensastegui) [inline src/main/webapp/js/navigation-loader.js:71]: For this function I think your method comment should only need to mention that the map will be accessed via a link

**Review by alanensastegui — CHANGES_REQUESTED** [review]
Would you mind commenting createMap and addMap. Here's Google's js conventions for reference https://google.github.io/styleguide/jsguide.html#jsdoc  I understand the two methods are pretty self explanatory, but I think it'd be helpful to explain the properties of the map created, and where the map is added to incase someone else ever wants to come back and edit/expand on your work in the future.  Other than that it looks good to merge

File: src/main/webapp/js/map_loader.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+let map;
+function createMap(){
+  <!--create map object and place in map element-->
+  map = new google.maps.Map(document.getElementById('map'), {
+    center: {lat: 37.422, lng: -122.084},
+    zoom: 16
+  });
+
+<!--add T-rex marker-->
```
Comment (Tetheta) [inline src/main/webapp/js/map_loader.js:9]: Just as an fyi, for comments in javascript we use // for line comments, and /* text here */ for block comments. The comment style here is supposed to be for html, bit surprised it works tbh

**Review by alanensastegui — APPROVED** [review]

File: src/main/webapp/js/map_loader.js (hunk 3 above)
Comment (faesalm) [reply src/main/webapp/js/map_loader.js:9]: oh interesting. It's just what the IDE defaulted to. fixed!

**Comment by alanensastegui** [comment]
LGTM

**Pull request merged**